### DISCUSSION TITLE
Error: Could not find a format to read the specified file in mode 'i'

### DISCUSSION THREAD

**Discussion by ahemberger** [colour-science/colour#984] [Q&A]
Like it says on the tin:

I have an EXR generated by Houdini Karma, which I'm trying to read with colour:

> `img_data = colour.read_image(evaluation_image)`

I get this error:

> `Error: Could not find a format to read the specified file in mode 'i'`

I'm on Windows 10, in Houdini, which runs Python 3.7.4:

>>> colour.utilities.describe_environment()
```
===============================================================================
*                                                                             *
*   Interpreter :                                                             *
*       python : 3.7.4 (1 file(s) copied., Feb 19 2022, 19:27:01) [MSC        *
*   v.1929 64 bit (AMD64)]                                                    *
*                                                                             *
*   colour-science.org :                                                      *
*       colour : 0.3.16                                                       *
*                                                                             *
*   Runtime :                                                                 *
*       imageio : 2.6.1                                                       *
*       networkx : 2.6.3                                                      *
*       numpy : 1.19.5                                                        *
*       scipy : 1.7.3                                                         *
*       six : 1.13.0                                                          *
*                                                                             *
===============================================================================
```

I notice when I first launch Houdini and import colour, I see:

`C:\Users\Allen\AppData\Roaming\Python\Python37\site-packages\colour\utilities\verbose.py:237: ColourUsageWarning: "OpenImageIO" related API features are not available, switching to "Imageio"!`

though I'm unsure this is related. I'm curious if anyone might have some hints how best to troubleshoot? This seemed to work totally fine on a macbook M1 with similar versions of everything.

**Comment by thomasmansencal** [comment]
Hi @ahemberger,

The issue is that Houdini does not have the *freeimage* backend, you can try the following in Houdini's Python interpreter:

```python
import imageio
imageio.plugins.freeimage.download()
```
